hapi/src/cron-jobs/monthly-transaction.js

```diff
@@ -0,0 +1,5 @@
+function sayHello() {
```

Comment:
Testing purpose

Reply:
If the purpose of this PR is testing something, why not lowering the cronjob time from a month to something more visible, like 10 minutes?

Reply:
@leisterfrancisco ups the montly-transaction name confused me :)